buildspec_publish_sar.yml

```diff
@@ -24,14 +24,22 @@ phases:
       - env
       - npm install
       - pip3 install aws-sam-cli
+      - apt install -q -y jq
+
   build:
     commands:
       - npm run build
   post_build:
     commands:
-      - sam package --template-file ./templates/sar_application_template.yml --s3-bucket $S3_BUCKET --output-template-file sampackaged.yaml
-      - sam publish --template sampackaged.yaml
-
+      - sam package --template-file ./templates/sar_application_template.yml --s3-bucket $S3_BUCKET --output-template-file sampackaged_raw.yaml
+      # Setting a timestamp for the build in labels
+      - BUILD_TIMESTAMP=$(date -Iseconds)
```

Comment:
Tror muligens vi kunne brukt CODEBUILD_START_TIME for å unngå en ekstra linje, men det er ikke så viktig antakelig.

Reply:
CODEBUILD_START_TIME inneholder space som labels ikke takler :-( Og det som er enda mer morsomt er at det er noe python involvert i tolkningen, slikt at det blir en dato og ikke en string - og da krasjer det!